src/wolframite/base/package.clj

```diff
@@ -0,0 +1,49 @@
+(ns wolframite.base.package
+  "The place to be for loading and manipulating Wolfram packages.
+"
+  (:require
+   [clojure.string :as string]
+   [wolframite.core :as wl]
+   [wolframite.impl.wolfram-syms.intern :as wi]
+   [wolframite.wolfram :as w]))
+
+(defn load-symbols
+  "Finds all of the public symbols in the given context and interns them to a clojure-accesible namespace. This is put under the given symbol or the context by default.
+
+  N.B. context does not contain the trailing '`'"
+  ([context]
+   (load-symbols context (symbol context)))
+  ([context sym--ns]
+   (let [names (wl/eval (w/Names (str context "`*")))
+         docs (mapv (fn [name]
+                      (mapv (fn [attr] (wl/eval (w/Information name attr)))
+                            ["FullName" "Usage"]))
+                    names)]
```

Comment:
a `for` may be nicer than 2 nested maps, but YMMV

Reply:
done.